opentelemetry_collector/receiver/vmimageagereceiver/factory.go

```diff
@@ -35,6 +39,8 @@ func (f *Factory) CreateDefaultConfig() configmodels.Receiver {
 	}
 }
 
+// CreateTraceReceiver generates an error because this receiver does not
```

Comment:
Is CreateTraceReceiver necessary if it always return an error?

Reply:
It's required to match the receiver Factory interface. https://github.com/open-telemetry/opentelemetry-collector/blob/4da23d11f0d8e3fa3309ac3804c9421b95235f32/receiver/factory.go#L63